test/Atmos/EDMF/ekman_layer.jl

```diff
@@ -104,19 +106,21 @@ function main(::Type{FT}, cl_args) where {FT}
         N_quad,
         param_set,
         surface = NeutralDrySurfaceModel{FT}(param_set),
+        coupling = Coupled(),
     )
 
     C_smag_ = C_smag(param_set)
-    turbulence = ConstantKinematicViscosity(FT(0.1))
+    turbulence = ConstantKinematicViscosity(FT(0))
```

Comment:
Following the comment above, I would keep this as it was. We can introduce another test for coupled EDMF.